.github/templates/main.yml

```diff
@@ -72,6 +73,20 @@ steps:
       retention-days: 1
 #@ end
 
+#@ def buildNugetPkgs():
+#@ for pkgName in nugetPackages:
+#@ buildCmd = "powershell MsBuild.exe ${{ github.workspace }}/Realm/" + pkgName + " -t:Pack -p:Configuration=Release -p:PackageOutputPath=${{ github.workspace }}/Realm/packages -restore"
+#@ stepName = "Build nuget package " + pkgName
+  - name: #@ stepName
+    run: #@ buildCmd
+#@ end
+  - name: Store artifacts
+    uses: actions/upload-artifact@v2
+    with:
+      path: ${{ github.workspace }}/Realm/packages/**
+      retention-days: 1
```

Comment:
No need to do it now, but we should perhaps tweak that for master vs PR builds. Maybe something like `{{ github.event_name != 'pull_request' && 30 || 1 }}`